src/trans/codegen_c.cpp

```diff
@@ -1210,7 +1210,9 @@ namespace {
                     {
                         args.push_back(c);
                     }
+#if defined(__APPLE__)
```

Comment:
Shouldn't this be `!defined`?

Reply:
Good catch. Fixed this locally when testing, but missed committing it.